smac/smbo/acquisition.py

```diff
@@ -208,12 +183,9 @@ def __init__(self,
 
         Parameters
         ----------
-        model : Model object
+        model : AbstractEPM
             A model that implements at least
-                 - predict(X)
-                 - getCurrentBestX().
-            If you want to calculate derivatives than it should also support
-                 - predictive_gradients(X)
+                 - predict_marginalized_over_instances(X)
```

Comment:
I would assume the same. @mfeurer added EIPS, so I'll ask him to add documentation.